src/features/wav_iterator.py

```diff
@@ -12,6 +12,7 @@
 from scipy.io import wavfile
 
 from . import spectral_features
+from . import mixer 
```

Comment:
This introduces a circular import (mixer imports from wav_iterator) and shouldn't be necessary, see below